subiquity/cloudinit.py

```diff
@@ -86,3 +154,23 @@ async def cloud_init_status_wait() -> (bool, Optional[str]):
     else:
         status = read_legacy_status(sp.stdout)
     return (True, status)
+
+
+async def validate_cloud_init_schema() -> bool:
```

Comment:
Maybe make the function return None since we're going to ignore the return value anyway?